File: Shapes.cpp
#include <iostream>
#include "Cube.hpp"
#include "Sphere.hpp"

int main() {
    int dimensions[] = {3, 5, 8};
    double length = 4.0;
    double radius = 4.0;

    for (int dim : dimensions) {
        Cube cube(dim, length);
        cube.Print();

        Sphere sphere(dim, radius);
        sphere.Print();
    }

    return 0;
}

File: shapes.py
from cube import Cube
from sphere import Sphere

def main():
    dimensions = [3, 5, 8]
    length = 4.0
    radius = 4.0

    for dim in dimensions:
        cube = Cube(ndim=dim, length=length)
        cube.print_info()

        sphere = Sphere(ndim=dim, radius=radius)
        sphere.print_info()

if __name__ == "__main__":
    main()
